### ISSUE TITLE
[Minor Issue] How to ignore accents

### ISSUE THREAD

**Issue by gatoborrachon** [mezz/JustEnoughItems#2349]
In other languares (like spanish) when you try to search something that has an accent (like "Tubería" which means Pipe, from BuildCraft) JEI doesn't recognize "Tuberia" (without the accent in the 'i') and throws nothing.

I guess that there's no config config to "ignore accents" so i wanted to suggest it, if its no a nuisance.
(also, if its possible to add this config to 1.12 as well, that would be great)

Many thanks for reading.

**Comment by mezz** [comment]
Thanks for the request!
There is currently no way to ignore accents. JEI would need to understand how to convert all accented characters into ASCII to add to the search. I am not sure how I would handle this by default, I suppose it may make sense to strip all accents by default on searches?

**Comment by gatoborrachon** [comment]
(so sorry for late reply, but i were busy these days)

> I suppose it may make sense to strip all accents by default on searches?

(sorry if i understood wrong) you mean to delete accented vowels from the searched items? (and then replace them with the same unaccented vowel)

**Comment by mezz** [comment]
Yes, one approach I can take is to convert all accented letters `ïîíī` to `i`. All `ń` converted to `n`. Etc. They would be displayed normally, but for searching it will treat the letters like there is no accent.
I think this may cause some minor issues where words are the same except with a different accent on a letter, but that is probably very rare.

**Comment by gatoborrachon** [comment]
> They would be displayed normally, but for searching it will treat the letters like there is no accent.

oh, nice

-----------------------

> I think this may cause some minor issues where words are the same except with a different accent on a letter, but that is probably very rare.

right, but as you say, that would be pretty rare (also, for practical uses, even if that happens, it wouldn't be an issue)

**Comment by mezz** [comment]
I think this is a duplicate of:
* https://github.com/mezz/JustEnoughItems/issues/1539
